user/tests.py

```diff
@@ -102,7 +99,119 @@ def test_signinview_post_fail_INVALID_PASSWORD(self):
             'phone_number'  : '010-1234-1234',
         }
 
-        response        = client.post('/users/signin', json.dumps(user), content_type='application/json')
+        response = client.post('/users/signin', json.dumps(user), content_type='application/json')
         self.assertEqual(response.status_code, 400)
         self.assertEqual(response.json(), {'MESSAGE' : 'INVALID_EMAIL_OR_PASSWORD'})
 
+class UserPasswordCheckTest(TestCase):
+    def setUp(self):
+        password        = '123qwe!@#QWE'
+        hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+        user = User(
+            email         = 'kst6294@gmail.com',
+            password      = hashed_password,
+            date_of_birth = '1991-09-04',
+            phone_number  = '010-1234-1234',
+        )
+
+        user.save()
+
+        self.access_token = jwt.encode({'id' : User.objects.get(id=user.id).id}, SECRET_KEY, algorithm=ALGORITHM)
+        self.user_id      = user.id
+
+    def tearDown(self):
+        User.objects.all().delete()
+
+    def test_password_is_same_check_post_success(self):
+        client = Client()
+        client = Client(HTTP_Authorization = self.access_token)
+
+        user = {
+            'user_id' : self.user_id,
+            'password' : '123qwe!@#QWE',
+        }
+
+        response = client.post('/users/password/check', json.dumps(user), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json(), {'MESSAGE' : 'SUCCESS'})
+
+    def test_password_is_same_check_post_fail(self):
+        client = Client()
+        client = Client(HTTP_Authorization = self.access_token)
+
+        user = {
+            'user_id' : self.user_id,
+            'password' : '23eewre!@#QWE',
+        }
+
+        response = client.post('/users/password/check', json.dumps(user), content_type='application/json')
```

Comment:
가독성을 위해서 `headers`를 post 인자로 전달해주세요~
```suggestion
        response = client.post('/users/password/check', json.dumps(user), content_type='application/json', headers=...)
```

Reply:
반영했습니다.

이런 좋은 방법을....